README.md

```diff
@@ -22,57 +30,92 @@
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-# 📖 [your_project_name] <a name="about-project">Hello Microverse</a>
 
-**[your_project__name]** This is a basic HTML & CSS project to practice all the tools that a wiil need to use in the future.
+# 📖 [Hello Microverse project] <a name="about-project"></a>
+
+
+**[Hello Microverse project]** is a basic Html and Css project made to practice with all the tools we will need ahead in the program. 
 
 ## 🛠 Built With <a name="built-with"></a>
 
+HTML5, CSS, Git and Github
 ### Tech Stack <a name="tech-stack"></a>
 
+
```

Comment:
- [ ] Your commit messages should be in imperative mood. Check [this article](https://www.theserverside.com/video/Follow-these-git-commit-message-guidelines#:~:text=If%20you%20want%20to%20write,or%20what%20you%20are%20doing.) to learn more.

 Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);